src/coreclr/tools/Common/Internal/Runtime/TargetFeatures.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Internal.TypeSystem;
+
+namespace Internal.Runtime
+{
+    internal static class TargetFeatures
```

Comment:
Is there an existing place to put this sort of feature flag?

Reply:
I don't know if it's worth it to abstract it like this. The codepath this is guarding is checking for library `/usr/lib/libobjc.dylib`. That's an OSX thing anyway.

Reply:
Yeah, it seemed like more trouble than it was worth. I reverted it.